1.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Depot</title>
+    <link rel="stylesheet" href="01.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">

```

Comment:
Is it okay to use bootstrap code to decorate the page E.g. mx-auto,d-clock . @PasinduUd 
@DevinDeSilva can you ask from the Mrs.Surangika about bootstrap ,plzzz

Reply:
Okay will ask and get back